postgis/geography_measurement.c

```diff
@@ -691,9 +691,12 @@ Datum geography_point_outside(PG_FUNCTION_ARGS)
 	/* Get an exterior point, based on this gbox */
 	gbox_pt_outside(&gbox, &pt);
 
-	lwpoint = lwpoint_make2d(4326, pt.x, pt.y);
-
-	g_out = gserialized_from_lwgeom((LWGEOM*)lwpoint, 1, &g_out_size);
+	lwpoint = (LWGEOM*) lwpoint_make2d(4326, pt.x, pt.y);
+	/* TODO: Investigate where this is used, this was probably not 
```

Comment:
It didn't matter what type it was returning, it was used for debugging of point-in-polygon code, so the output was just being turned into text to inspect.